src/Repository/TrickRepository.php

```diff
@@ -19,28 +19,23 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Trick::class);
     }
 
-    // /**
```

Comment:
you could delete all the commented code in your repo

Reply:
Fixed in Corrections branch